### PR TITLE
Update Paradice.in.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
Added-New-Features

### CODE REVIEW

**Review by faucetcollector — DISMISSED** [review]

File: Paradice.in/Paradice.in.fc [hunk 1]
```diff
@@ -275,37 +255,84 @@ public class Paradice : FaucetScript
...
+		}
+		
+        if (!IsVisible(DailyBonus_Check))
+        {
+         Log("DailyBonus - Available - do stuff ");	
+         var Claim_Button = ElementByXPath("//div[contains(@class,'modal-content')]//span[contains(text(),'Claim')]");
+		 Click(Claim_Button);
+		 Wait(10);
+         var SuccessMessage = ElementByXPath("//span[contains(@class,'two')]//span");
+         Log("DailyBonus - Congratulations, You have received " + SuccessMessage.Text + " SATOSHI" );
```
[the remark this answers is not in the document]
Comment (aviramdayan) [reply Paradice.in/Paradice.in.fc:282]: yeah need to add if IsVisible

File: Paradice.in/Paradice.in.fc [hunk 2]
```diff
@@ -275,37 +255,84 @@ public class Paradice : FaucetScript
...
+	    Wait();
+	    Wait(10);
+
+        var DailyBonus_Check = ElementByXPath("//span[contains(@class,'one') and contains(text(),'already received') ]"); //check
+        if (IsVisible(DailyBonus_Check))
+        {
+         Log("DailyBonus - already received - try tomorrow");
+		}
+		
+        if (!IsVisible(DailyBonus_Check))
```
[the remark this answers is not in the document]
Comment (aviramdayan) [reply Paradice.in/Paradice.in.fc:275]: i prefer !IsVisible

File: Paradice.in/Paradice.in.fc [hunk 3]
```diff
@@ -275,37 +255,84 @@ public class Paradice : FaucetScript
...
+		}
+		
+        if (!IsVisible(LuckySpin_Check))
+        {
+         Log("LuckySpin - Available - do stuff ");	
+         var Claim_Button = ElementByXPath("//div[contains(@class,'modal-content')]//span[contains(text(),'Spin')]");
+		 Click(Claim_Button);
+		 Wait(10);
+         var SuccessMessage = ElementByXPath("//span[contains(@class,'win-amount')]");
+         Log("LuckySpin - Congratulations, You have received " + SuccessMessage.Text);
```
[the remark this answers is not in the document]
Comment (aviramdayan) [reply Paradice.in/Paradice.in.fc:313]: yeah need to add if IsVisible

**Comment by aviramdayan** [comment]
you can approve here

**Review by cgerabit — APPROVED** [review]

**Pull request merged**